modules/oe_whitelabel_multilingual/config/optional/block.block.openeuropa_content_language_switcher.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
```

Comment:
move this file into oe_whitelabel/config/optional with the rest of the block config exports

Reply:
If we move this file and rename it  as on the suggestion above it will replace the already existing block placed on the nav_right region.
How about renaming it to oe_whitelabel_MULTILINGUAL_language_switcher instead?